pkgs/development/tools/rust/bindgen/unwrapped.nix

```diff
@@ -7,15 +7,15 @@ let
   rustfmt-nightly = rustfmt.override { asNightly = true; };
 in rustPlatform.buildRustPackage rec {
   pname = "rust-bindgen-unwrapped";
-  version = "0.66.1";
+  version = "0.69.1";
```

Comment:
Please break this version bump out into a separate PR.

Reply:
I've opened #271407.